core/node/vm_runner/src/output_handler.rs

```diff
@@ -173,7 +173,10 @@ impl StateKeeperOutputHandler for AsyncOutputHandler {
             } => {
                 sender
                     .send(tokio::task::spawn(async move {
-                        handler.handle_l1_batch(updates_manager).await
+                        let started_at = Instant::now();
+                        let result = handler.handle_l1_batch(updates_manager).await;
+                        METRICS.output_handle_time.observe(started_at.elapsed());
```

Comment:
Nit: You can use [`Histogram::start()`](https://matter-labs.github.io/vise/vise/struct.Histogram.html#method.start) / `observe()` to avoid dealing with `Instant`s.